### PR TITLE
fix: add missing error checking and fix linting

### PR DESCRIPTION
**Pull request by frrist**
- somehow this wasn't caught in CI or in my last local run of `git commit`. Werid.

### CODE REVIEW

File: cmd/cli/cancel/cancel.go [hunk 1]
```diff
@@ -141,10 +141,10 @@ func cancel(cmd *cobra.Command, cmdArgs []string, options *CancelOptions) error
 	if err != nil {
 		spinner.Done(printer.StopFailed)
 
-		if err, ok := err.(*bacerrors.ErrorResponse); ok {
-			return err
+		if errResp, ok := err.(*bacerrors.ErrorResponse); ok {
+			return errResp
 		} else {
-			return fmt.Errorf("unknown error trying to cancel job (ID: %s): %+v", requestedJobID, err)
+			return fmt.Errorf("unknown error trying to cancel job (ID: %s): %+v", requestedJobID, errResp)
```
Comment (rossjones) [inline cmd/cli/cancel/cancel.go:147]: Shouldn't this be `err` if !ok then type assertion failed.

File: cmd/cli/describe/describe.go [hunk 2]
```diff
@@ -101,10 +101,10 @@ func describe(cmd *cobra.Command, cmdArgs []string, OD *DescribeOptions) error {
 	j, foundJob, err := util.GetAPIClient(ctx).Get(ctx, inputJobID)
 
 	if err != nil {
-		if err, ok := err.(*bacerrors.ErrorResponse); ok {
-			return err
+		if errResp, ok := err.(*bacerrors.ErrorResponse); ok {
+			return errResp
 		} else {
-			return fmt.Errorf("unknown error trying to get job (ID: %s): %w", inputJobID, err)
+			return fmt.Errorf("unknown error trying to get job (ID: %s): %w", inputJobID, errResp)
```
Comment (rossjones) [inline cmd/cli/describe/describe.go:107]: As per previous comment.

File: cmd/util/logging.go [hunk 3]
```diff
@@ -67,10 +67,10 @@ func Logs(cmd *cobra.Command, jobID string, follow, history bool) error {
 	// any dataframes that are logged from the requested execution/job.
 	conn, err := apiClient.Logs(ctx, jobID, executionID, history, follow)
 	if err != nil {
-		if err, ok := err.(*bacerrors.ErrorResponse); ok {
-			return err
+		if errResp, ok := err.(*bacerrors.ErrorResponse); ok {
+			return errResp
 		} else {
-			return fmt.Errorf("unknown error trying to stream logs from job (ID: %s): %w", requestedJobID, err)
+			return fmt.Errorf("unknown error trying to stream logs from job (ID: %s): %w", requestedJobID, errResp)
```
Comment (rossjones) [inline cmd/util/logging.go:73]: As per previous comments.

**Review by rossjones — APPROVED** [review]
Couple of minor comments about use of err after type assertion.

File: cmd/cli/cancel/cancel.go (hunk 1 above)
Comment (frrist) [reply cmd/cli/cancel/cancel.go:147]: ahh wow, yes 🤦

**Pull request merged**